lib/nimble.phx.gen.template/addons/variants/web/wallaby.ex

```diff
@@ -0,0 +1,194 @@
+defmodule Nimble.Phx.Gen.Template.Addons.Web.Wallaby do
+  use Nimble.Phx.Gen.Template.Addon
+
+  @versions %{
+    wallaby: "~> 0.26.2",
+    otp_version: "23.0.2",
+    elixir_version: "1.10.4"
+  }
+
+  @impl true
+  def do_apply(%Project{} = project, _opts) do
+    project
+    |> copy_files
+    |> edit_files
+  end
+
+  defp copy_files(%Project{} = project) do
+    binding = [
+      web_module: project.web_module,
+      base_module: project.base_module
+    ]
+
+    files = [
+      {:eex, "test/support/feature_case.ex.eex", "test/support/feature_case.ex"},
+      {:eex, "test/features/home_page/view_home_page_test.exs.eex",
+       "#{project.web_test_path}/features/home_page/view_home_page_test.exs"}
+    ]
+
+    Generator.copy_file(files, binding)
+
+    if File.exists?(Path.join([".github", "workflows", "test.yml"])) do
+      copy_github_action()
```

Comment:
should this be in a Github addon? and in Github addon, we check if the variant is a `web` then we use `test.yml` web version.